### PR TITLE
add ERD

### PR DESCRIPTION
**Pull request by iam-Akshat**
- Add ERD diagram for the  app

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @iam-Akshat :wave: 

You have a nice ERD diagram but there are some changes to be made before this milestone can be approved.

## REQUIRED

* [x] In your comment and post table, **content** is missing.
* [x] You missed out the **name** in the user table

### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done well so far. Everything looks good except for one thing :blush: 
Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Since `send_friend` and `receive_friend` are both instances of the user model, there should be an inverse relationship between the user and the friendship model. This implies that a user can send many friend requests and can also receive many friend requests, but a friend request, either sent or received, can belong to just only one user. See the diagram below for details:
rel



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by mimipeshy** [comment]
### STATUS :APPROVED :partying_face: 

Hey there @iam-Akshat  :wave: :wave:  good job on working on the requested changes. This project is therefore approved. Go ahead and merge. All the best in the next milestones :man_technologist: 

Regards
Peris Ndanu

**Pull request merged**
